src/user-guide/running-workflows/task-job-states.rst

```diff
@@ -0,0 +1,79 @@
+Task and Job States
+===================
+
+**Tasks** are a workflow abstraction; they represent future and past jobs as
+well as current active ones.
+
+**Jobs** are less of an abstraction; they represent real job scripts submitted
+to run as a process on a computer somewhere, or the final status of those
+real processes.
+
+A single task can submit multiple jobs, by automatic retry or manual triggering.
+
+In the Cylc UI, task states are represented by monochromatic task icons, and
+job states by coloured job icons.
+
+
+.. TODO include task and job state images 
```

Comment:
Note, I think we can use the Cylc UI test framework (Cypress) to auto-generate screenshots of UI components e.g. the task/job state key when we get the time.